### PR TITLE
[MNG-7661] Remove deprecated 'verifier.resetStreams()' 

### PR DESCRIPTION
**Pull request by psiroky**
* I know the change looks huge, but in fact it is a find & replace call in IDE. 99% were of the changes were done automatically, just had to manually adjust some edge cases

### CODE REVIEW

File: core-it-suite/src/test/java/org/apache/maven/it/MavenITmng2199ParentVersionRangeTest.java [hunk 1]
```diff
@@ -65,7 +65,6 @@ public void testValidParentVersionRangeWithInclusiveUpperBound()
         {
             if ( verifier != null )
             {
-                verifier.resetStreams();
             }
         }
```
Comment (slawekjaranowski) [inline core-it-suite/src/test/java/org/apache/maven/it/MavenITmng2199ParentVersionRangeTest.java:70]: empty if and finally ?

**Comment by slawekjaranowski** [comment]
Great job, can you split each commit as separate PR 😄

File: core-it-suite/src/test/java/org/apache/maven/it/MavenITmng1803PomValidationErrorIncludesLineNumberTest.java [hunk 2]
```diff
@@ -64,7 +64,6 @@ public void testit()
         }
         finally
         {
-            verifier.resetStreams();
         }
```
[the remark this answers is not in the document]
Comment (psiroky) [reply core-it-suite/src/test/java/org/apache/maven/it/MavenITmng1803PomValidationErrorIncludesLineNumberTest.java]: Fixed, thanks for spotting this.

**Comment by psiroky** [comment]
I will leave this PR for the first change (remvoing `resetStreams()`) and will create two more PR with the other changes.

File: core-it-suite/src/test/java/org/apache/maven/it/MavenITmng2199ParentVersionRangeTest.java [hunk 3]
```diff
@@ -100,7 +89,6 @@ public void testValidParentVersionRangeWithExclusiveUpperBound()
         {
             if ( verifier != null )
             {
-                verifier.resetStreams();
```
Comment (psiroky) [inline core-it-suite/src/test/java/org/apache/maven/it/MavenITmng2199ParentVersionRangeTest.java:103]: Hmm, there is more of these here :facepalm: . Will fix those as well.

File: core-it-suite/src/test/java/org/apache/maven/it/MavenITmng3331ModulePathNormalizationTest.java [hunk 4]
```diff
@@ -68,7 +68,6 @@ public void testitMNG3331a ()
          * Reset the streams before executing the verifier
```
Comment (psiroky) [inline core-it-suite/src/test/java/org/apache/maven/it/MavenITmng3331ModulePathNormalizationTest.java:68]: Removing these comments as well...

**Comment by psiroky** [comment]
I went through the diff (as best as I could considering the number of changed files), found couple more discrepancies and fixed them. Please take another look. Hopefully I got them all.

File: core-it-suite/src/test/java/org/apache/maven/it/ItUtils.java [hunk 5]
```diff
@@ -19,7 +19,6 @@
  * under the License.
  */
 
-import org.junit.jupiter.api.Test;
```
Comment (psiroky) [inline core-it-suite/src/test/java/org/apache/maven/it/ItUtils.java:22]: This is unrelated, but it's an unused import, so hopefully fine to remove it as part of this.
